stable/syncthing/SCALE/questions.yaml

```diff
@@ -141,17 +141,120 @@ questions:
                       schema:
                         type: int
                         default: 8384
-                        editable: false
+                        editable: true
+                        hidden: false
+                    - variable: nodePort
+                      label: "Host nodePort to expose to (optional)"
+                      description: "Only gets used when nodePort is selected"
+                      schema:
+                        type: int
+                        min: 9000
+                        max: 65535
+                        default: 36024
```

Comment:
Did you read the list of default nodeports in the manual @stavros-k  made?

Reply:
Yes, this is the correct one.

Reply:
Is see, it was just missing it seems...